app/src/main/java/pm/gnosis/heimdall/ui/onboarding/account/AccountSetupViewModel.kt

```diff
@@ -15,7 +15,7 @@ import javax.inject.Inject
 class AccountSetupViewModel @Inject constructor(
     private val accountsRepository: AccountsRepository,
     private val bip39: Bip39,
-    private val googleSmartLockRepository: GoogleSmartLockRepository
+    private val googleSmartLockRepository: GoogleSmartLockHelper
```

Comment:
How can we test this now? Or do we change that once we say it is final and we want to add tests?

Reply:
Yeah let's see if it is final or not. The flow is way too specific in my opinion to have a general store credentials otherwise.